templates/starter/terraform/README.md

```diff
@@ -0,0 +1,9 @@
+# Azure Developer CLI (azd) Terraform Starter
+
+A starter template with [Terraform](https://aka.ms/azure-dev/terraform) as infrastructure provider for [Azure Developer CLI](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/overview) (azd).
```

Comment:
A suggestion from @vhvb1989:

> I would also add a section below with `## next steps` to list what should a user try to do next... like create applications or how to connect source code with infra...

Reply:
I agree about a next steps section. Possible steps to include: 
- connecting source code with infra
- adding to / modifying the infra to fit their needs

Reply:
Basically, the premise is that we're writing the entire specification of how a user builds an azd application from scratch, so it may be pertinent to have closer eyes.